src/Java.Interop.Tools.Generator/Metadata/NamespaceTransform.cs

```diff
@@ -0,0 +1,123 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Xml.Linq;
+using Xamarin.Android.Tools;
+
+namespace Java.Interop.Tools.Generator
+{
+	public class NamespaceTransform
```

Comment:
Rando aside: would this be a good use case for C#9 `records`? https://docs.microsoft.com/en-us/dotnet/csharp/whats-new/csharp-9#record-types

Reply:
Doesn't look like you can override primary constructors (?!), so this *isn't* a good use case.

Reply:
However, you can still use `record` types *without* using primary constructors…

Reply:
Reading through the link, I don't see any benefits to using `record`, and we haven't bumped to C# 9 yet.